kafka_quota_enforcer/src/main/java/com/tesla/data/quota/enforcer/AdminClientQuotaService.java

```diff
@@ -60,8 +60,8 @@ public void create(Collection<ConfiguredQuota> quotas) {
       // For unspecified quota types, ConfigureQuota returns null. We use such null value directly in
       // ClientQuotaAlternation.Op to clear the quota types in the cluster.
       Collection<ClientQuotaAlteration.Op> ops = List.of(
-          new ClientQuotaAlteration.Op(PRODUCER_BYTE_RATE, q.getProducerByteRate()),
-          new ClientQuotaAlteration.Op(CONSUMER_BYTE_RATE, q.getConsumerByteRate()),
+          new ClientQuotaAlteration.Op(PRODUCER_BYTE_RATE, q.getProducerByteRate() != null ? q.getProducerByteRate().doubleValue() : null),
+          new ClientQuotaAlteration.Op(CONSUMER_BYTE_RATE, q.getConsumerByteRate() != null ? q.getConsumerByteRate().doubleValue() : null),
```

Comment:
nit: Should we create a helper method in `ConfiguredQuota` to wrap this logic?
```
public static <T extends Number, R> R mapIfNotNull(T value, Function<T, R> mapper) {
    return value != null ? mapper.apply(value) : null;
}
```
Now it can be used with
```
mapIfNotNull(q.getProducerByteRate(), Number::doubleValue);
mapIfNotNull(quotas.get(CONSUMER_BYTE_RATE), Number::intValue);
```

Reply:
Also, how come we get the double value here and int value below? Why not just get the int value everywhere if we need to truncate anyway?

Reply:
> Also, how come we get the double value here and int value below? Why not just get the int value everywhere if we need to truncate anyway?

Well because the request percentage takes fractional values. 

So this PR effectively makes the double / integer conversion implicitly, which probably causes confusions to our users. Instead, I'm consider dropping this PR and just letting those values be rejected by Kafka APIs and fail, so the users would know values are bad. WDTY?